.github/workflows/test.yaml

```diff
@@ -30,23 +47,25 @@ jobs:
       with:
         path: /var/cache/apt
         key: var-cache-apt-{{ hashFiles('**/debian/control') }}
-    - name: Run "apt update"
-      run: |
-          sudo apt update
     - name: Download Debian dependencies
       if: steps.cache-deb-downloads.outputs.cache-hit != 'true'
       run: |
           sudo apt clean
+          sudo apt update
           sudo apt build-dep -d -y ${{ github.workspace }}/src/github.com/snapcore/snapd
     # Work around caching files owned by root https://github.com/actions/cache/issues/133
```

Comment:
it seems that issue was closed as well? https://github.com/actions/cache/issues/133 Not a blocker for this PR but something to look into perhaps?

Reply:
Thanks for spotting this. I agree we should remove it and see if it works now.

Reply:
I moved it back.